README.md

```diff
@@ -213,26 +213,29 @@ Short backend is built on top of
 [Uncle Bob's Clean Architecture](https://api.short-d.com/r/ca), the central
 objective of which is separation of concerns.
 
-![Clean Architecture](doc/eng/clean-architecture/clean-architecture.jpg)
-![Boundary](doc/eng/clean-architecture/boundary.jpg)
+![Clean Architecture](doc/clean-architecture/clean-architecture.jpg)
+
+![Short Backend](doc/clean-architecture/short-backend.jpg)
 
 It enables the developers to modify a single component of the system at a time
 while leaving the rest unchanged. This minimizes the amount of changes have to
 be made in order to support new requirements as the system grows. Clean
 Architecture also improves the testability of system, which in turn saves
 precious time when creating automated tests.
 
-Here is an example of finance app using clean architecture:
-
-![Finance App](doc/eng/clean-architecture/finance-app.jpg)
-
 ### Service Level Architecture
 
 Short adopts [Microservices Architecture](https://api.short-d.com/r/ms) to
 organize dependent services around business capabilities and to enable
 independent deployment of each service.
 
-![Microservice Architecture](doc/eng/microservices.jpg)
+![Short Cloud](doc/cloud/overall.jpg)
+[SSR](https://docs.google.com/document/d/1TuWexeKwhQh8JTytRAwST3XujBi0wTGExwJan-WfXWs), 
+[Toggle](https://docs.google.com/document/d/1TuWexeKwhQh8JTytRAwST3XujBi0wTGExwJan-WfXWs),
+[Status Page](https://docs.google.com/document/d/1pgRNnD8yAlEmj-sucS_FZ89LdvBy5zpKQ9OvILoBqDM), Search,
```

Comment:
Just wanted to inform that the "Search" exists in the line with no docs and whether it is intentional.

Reply:
Yes, it is intended. No design doc for search yet.